README.md

```diff
@@ -48,159 +47,156 @@ limitations under the License.
     </a>
 </p>
 
-## Overview
+<h3> DeepSparse is a deep learning inference engine for running super-fast sparse models. 🚀🚀🚀</h3>
 
-The DeepSparse Engine is a CPU runtime that delivers GPU-class performance by taking advantage of sparsity (read more about sparsification [here](https://docs.neuralmagic.com/main/source/getstarted.html#sparsification)) within neural networks to reduce compute required as well as accelerate memory bound workloads. It is focused on model deployment and scaling machine learning pipelines, fitting seamlessly into your existing deployments as an inference backend.
+A CPU runtime engine delivering GPU-class performance by taking advantage of sparsity within neural networks to reduce compute required as well as accelerate memory bound workloads. Read more about sparsification [here](https://docs.neuralmagic.com/main/source/getstarted.html#sparsification).
```

Comment:
We should keep the original short description for now as these changes require a larger consensus. The final approved copy would need to be updated not just in our Naming Brief but in all the places that utilize this asset.